.gitattributes

```diff
@@ -1,5 +1,7 @@
 # Set the default behavior, in case people don't have core.autocrlf set.
-* text=auto
+* text=lf
+
+*.expected binary
```

Comment:
Does this stop .expected files from showing up in `git diff`? No, right?

How would this differ from just having `eol=lf` in this file?

Reply:
I suspect not.

But your second point makes sense and I think I'll just switch it to that. That should be sufficient.